java-manta-client/src/main/java/com/joyent/manta/client/crypto/SupportedCipherDetails.java

```diff
@@ -69,6 +69,14 @@
     Cipher getCipher();
 
     /**
+     * In some cases we <em>need</em> the Cipher to live entirely in Java.
```

Comment:
I don't feel great about this change, but I don't see a better way. @cburroughs may have some suggestions.